lib/usb-connection.js

```diff
@@ -23,7 +29,7 @@ try {
   // var VENDOR_REQ_IN  = usb.LIBUSB_REQUEST_TYPE_VENDOR | usb.LIBUSB_RECIPIENT_DEVICE | usb.LIBUSB_ENDPOINT_IN;
 } catch (e) {
   haveusb = false;
-  logs.err('WARNING: No usb controller found on this system.');
+  log.error('WARNING: No usb controller found on this system.');
 }
```

Comment:
Seems like an entirely unrelated problem that normal use can't possibly warrant handling for :P


that is... i think you changed node versions, npm installed, then changed back.